apps/build_stitched_index.cpp

```diff
@@ -285,7 +285,7 @@ void prune_and_save(path final_index_path_prefix, path full_index_path_prefix, p
     auto pruning_index_timer = std::chrono::high_resolution_clock::now();
 
     diskann::get_bin_metadata(input_data_path, number_of_label_points, dimension);
-    diskann::Index<T> index(diskann::Metric::L2, dimension, number_of_label_points, false, false);
+    diskann::Index<T> index(diskann::Metric::L2, dimension, number_of_label_points, nullptr, nullptr, 0, false, false);
```

Comment:
As above, should we change this to start using the new constructor?

Reply:
It is indeed using the new constructor. Its a matter of convenience, we have 2 constructors, one that takes config and the other takes args directly. If we use the constructor with config, we have to write too many lines of code to achieve this. not idea i think.